code/datums/storage/storage.md

```diff
@@ -0,0 +1,43 @@
+
+# Storage datums
```

Comment:
Do you think you could move this folder to `code/modules`? Trying to clear out this folder eventually.

Reply:
Sure